acceptance/README.md

```diff
@@ -1,257 +1,467 @@
+      - apt-get update && apt-get install -y cron locales-all net-tools wget systemd-sysv
+        gnupg
+    hypervisor: docker
+
+    mount_folders:
+      puppet:
+        host_path: ~/puppet
+        container_path: /build/puppet
+    roles:
+      - agent
+CONFIG:
+  nfs_server: none
+  consoleport: 443
+  pooling_api: http://vmpooler.delivery.puppetlabs.net/
+```
+Run acceptance tests against pre-built puppet-agent packages with
+`bundle exec rake ci:test:aio SHA=<sha or tag> TESTS=path/to/test.rb HOSTS=hosts.yaml`
+
+When you generate your [hosts file](#hosts.yaml), [beaker-hostgenerator][] does
+its best to populate the values as logically as possible. You will likely want
+to update or modify them to suite your needs.
+
+With `image`, [beaker-hostgenerator][] does its best to guess the most logical
+image string based on the platform you are building. For the most part, this
+should work without interference, but if you are using a custom docker image or
+do not want the default, then you will have to manually update this string. The
+example above has modified `image` to pull from the pcr-internal docker repo.
+
+`docker_image_commands` is automatically populated when generating the hosts
+file with [beaker-hostgenerator][]. This has already been set for a handful of
+host types, but may not be set for all.
+
+* TODO I believe I had a few issues with the initial docker image setup. I'd
+    like to go through these steps with someone else so that I can remember what
+    that initial pain was exactly.
+* TODO I only tried it once with a docker master, but the image I used was SO
+    SLOW, so I gave up. I also ran out of time. So this will be something that
+    would be good to investigate more fully.
+* TODO check with Casey and Molly about these. I can't quite remember the
+    details, but I do remember that we found `docker_image_entrypoint`
+    preferable to `docker_cmd`.
+* TODO These docker containers have to run in priviledged mode (or systemd,
+    among possibly other things, won't function as we need them to). This is
+    not ideal if you're testing code that affects your OS (ie running docker on
+    linux without a docker machine in between the container and your laptop).
+* TODO add emphasis that you need an account and permissions to access the images
+    on pcr-internal.puppet.net
+
+#### Static Hosts
+To test on a server that's already been spun up or doesn't require a hypervisor,
+you should set the name of the host to the FQDN of the server you want to use,
+then remove the hypervisor and template settings. This is not recommended, and
+you may run into issues with failures or overwritten configuration due to either
+beaker provision steps or test provisioning steps.
+```
+---
+HOSTS:
+    azeqdqmk14mvu3g.delivery.puppetlabs.net:
+        platform: el-7-x86_64
+        packaging_platform: el-7-x86_64
+        roles:
+          - master
+```
+This is not recommended unless you are familiar with how [beaker][] and
```

Comment:
Start the section with this sentence to emphasize we're not recommending this option.